Description: Version 2.0.0
 Live folder mode, command line options for every setting and the new
 defaults (patches 16-19) make this a different program from upstream
 1.3.3, so it reports itself as MiniDLNA 2.0.0.
Forwarded: not-needed
Last-Update: 2026-10-21
--- a/configure.ac
+++ b/configure.ac
@@ -14,7 +14,7 @@
 # License along with MiniDLNA; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307
 # USA.
-AC_INIT([MiniDLNA],[1.3.3],[],[minidlna],[http://sourceforge.net/projects/minidlna/])
+AC_INIT([MiniDLNA],[2.0.0],[],[minidlna],[http://sourceforge.net/projects/minidlna/])
 #LT_INIT
 
 AC_CANONICAL_TARGET
--- a/upnpglobalvars.h
+++ b/upnpglobalvars.h
@@ -57,7 +57,7 @@
 
 #include <sqlite3.h>
 
-#define MINIDLNA_VERSION "1.3.3"
+#define MINIDLNA_VERSION "2.0.0"
 
 #ifdef NETGEAR
 # define SERVER_NAME "ReadyDLNA"
